c7n/policy.py

```diff
@@ -244,24 +244,31 @@ def run(self, *args, **kw):
             return
 
         with self.policy.ctx:
-            self.policy.log.info(
+            self.policy.log.debug(
                 "Running policy %s resource: %s region:%s c7n:%s",
                 self.policy.name, self.policy.resource_type,
                 self.policy.options.region or 'default',
                 version)
 
-            s = time.time()
-            resources = self.policy.resource_manager.resources()
-            rt = time.time() - s
+            region        = self.policy.options.region or 'default'
+            resource      = self.policy.resource_type
+            start         = time.time()
+            resources     = self.policy.resource_manager.resources()
+            run_time      = time.time() - start
+            qty_resources = str(len(resources))
             self.policy.log.info(
-                "policy: %s resource:%s has count:%d time:%0.2f" % (
+                "count: %s for policy: %s resource: %s region: %s time: %0.2f" % (
```

Comment:
I removed any ljust or rjust stuff from this section. let me know if there are any other changes you want on this PR